core-tests/shared/src/test/scala/zio/stm/ZSTMSpec.scala

```diff
@@ -35,18 +35,18 @@ object ZSTMSpec extends ZIOBaseSpec {
         }
       ),
       testM("`doWhile` to run effect while it satisfies predicate") {
-        for {
-          a <- TQueue.bounded[Int](5).commit
-          _ <- a.offerAll(List(0, 0, 0, 1, 2)).commit
-          n <- a.take.doWhile(_ == 0).commit
-        } yield assert(n)(equalTo(1))
+        (for {
```

Comment:
You can also define a local variable here if you don't want to have to put the parentheses around the for comprehension.

```
val transaction = for { ... }
assertM(transaction.commit)(???)
```